include/aspect/material_model/rheology/drucker_prager.h

```diff
@@ -76,9 +80,14 @@ namespace aspect
            *
            * @param [in] prm The ParameterHandler to read from.
            */
-          DruckerPragerParameters
-          parse_parameters (const unsigned int n_fields,
-                            ParameterHandler &prm);
+          void
+          parse_parameters (ParameterHandler &prm,
+                            const std::shared_ptr<std::vector<unsigned int>> &expected_n_phases_per_composition =
+                              std::shared_ptr<std::vector<unsigned int>>());
+          const DruckerPragerParameters
```

Comment:
one empty line between function definitions

Reply:
@elodie-kendall - can you add a space between this line and the next one?